diff --git a/node_modules/app-builder-lib/electron-osx-sign/util.js b/node_modules/app-builder-lib/electron-osx-sign/util.js
index 107d8aa..4a6a354 100644
--- a/node_modules/app-builder-lib/electron-osx-sign/util.js
+++ b/node_modules/app-builder-lib/electron-osx-sign/util.js
@@ -202,12 +202,23 @@ module.exports.walkAsync = async function (dirPath) {
       const stat = await fs.lstat(filePath)
       if (stat.isFile()) {
         switch (path.extname(filePath)) {
+          case '': // Binary
+           if (path.basename(filePath)[0] !== '.') {
+             return getFilePathIfBinaryAsync(filePath)
+           } // Else reject hidden file
+           break
+         case '.dylib': // Dynamic library
+         case '.node': // Native node addon
+           return filePath
           case '.cstemp': // Temporary file generated from past codesign
             debuglog('Removing... ' + filePath)
             await fs.unlink(filePath)
             return
           default:
-            return getFilePathIfBinaryAsync(filePath)
+            if (path.extname(filePath).indexOf(' ') >= 0) {
+               // Still consider the file as binary if extension seems invalid
+               return getFilePathIfBinaryAsync(filePath)
+             }
         }
       } else if (stat.isDirectory() && !stat.isSymbolicLink()) {
         const result = await _walkAsync(filePath)
