diff --git a/node_modules/dmg-builder/.DS_Store b/node_modules/dmg-builder/.DS_Store
new file mode 100644
index 0000000..ba74fd3
Binary files /dev/null and b/node_modules/dmg-builder/.DS_Store differ
diff --git a/node_modules/dmg-builder/out/dmg.js b/node_modules/dmg-builder/out/dmg.js
index ad0bb2e..4a64444 100644
--- a/node_modules/dmg-builder/out/dmg.js
+++ b/node_modules/dmg-builder/out/dmg.js
@@ -295,7 +295,8 @@ exports.DmgTarget = DmgTarget;
 async function createStageDmg(tempDmg, appPath, volumeName) {
   //noinspection SpellCheckingInspection
   const imageArgs = addLogLevel(["create", "-srcfolder", appPath, "-volname", volumeName, "-anyowners", "-nospotlight", "-format", "UDRW"]);
-  imageArgs.push("-fs", "HFS+", "-fsargs", "-c c=64,a=16,e=16");
+  //HFS+ is deprecated and broken on Big Sur
+  //imageArgs.push("-fs", "HFS+", "-fsargs", "-c c=64,a=16,e=16");
   imageArgs.push(tempDmg);
   await (0, _builderUtil().spawn)("hdiutil", imageArgs);
   return tempDmg;
diff --git a/node_modules/dmg-builder/vendor/.DS_Store b/node_modules/dmg-builder/vendor/.DS_Store
new file mode 100644
index 0000000..ca51389
Binary files /dev/null and b/node_modules/dmg-builder/vendor/.DS_Store differ
diff --git a/node_modules/dmg-builder/vendor/biplist/__init__.pyc b/node_modules/dmg-builder/vendor/biplist/__init__.pyc
index 61a18eb..8df6d1b 100644
Binary files a/node_modules/dmg-builder/vendor/biplist/__init__.pyc and b/node_modules/dmg-builder/vendor/biplist/__init__.pyc differ
diff --git a/node_modules/dmg-builder/vendor/dmgbuild/badge.pyc b/node_modules/dmg-builder/vendor/dmgbuild/badge.pyc
index 1f80125..063fd19 100644
Binary files a/node_modules/dmg-builder/vendor/dmgbuild/badge.pyc and b/node_modules/dmg-builder/vendor/dmgbuild/badge.pyc differ
diff --git a/node_modules/dmg-builder/vendor/dmgbuild/colors.pyc b/node_modules/dmg-builder/vendor/dmgbuild/colors.pyc
index 1beee78..4d93ec3 100644
Binary files a/node_modules/dmg-builder/vendor/dmgbuild/colors.pyc and b/node_modules/dmg-builder/vendor/dmgbuild/colors.pyc differ
diff --git a/node_modules/dmg-builder/vendor/ds_store/__init__.pyc b/node_modules/dmg-builder/vendor/ds_store/__init__.pyc
index 60fee57..83ebd2d 100644
Binary files a/node_modules/dmg-builder/vendor/ds_store/__init__.pyc and b/node_modules/dmg-builder/vendor/ds_store/__init__.pyc differ
diff --git a/node_modules/dmg-builder/vendor/ds_store/buddy.pyc b/node_modules/dmg-builder/vendor/ds_store/buddy.pyc
index f5df453..13a1584 100644
Binary files a/node_modules/dmg-builder/vendor/ds_store/buddy.pyc and b/node_modules/dmg-builder/vendor/ds_store/buddy.pyc differ
diff --git a/node_modules/dmg-builder/vendor/ds_store/store.pyc b/node_modules/dmg-builder/vendor/ds_store/store.pyc
index 6560639..4045fd7 100644
Binary files a/node_modules/dmg-builder/vendor/ds_store/store.pyc and b/node_modules/dmg-builder/vendor/ds_store/store.pyc differ
diff --git a/node_modules/dmg-builder/vendor/mac_alias/__init__.pyc b/node_modules/dmg-builder/vendor/mac_alias/__init__.pyc
index 88a4bf2..94400bd 100644
Binary files a/node_modules/dmg-builder/vendor/mac_alias/__init__.pyc and b/node_modules/dmg-builder/vendor/mac_alias/__init__.pyc differ
diff --git a/node_modules/dmg-builder/vendor/mac_alias/alias.pyc b/node_modules/dmg-builder/vendor/mac_alias/alias.pyc
index 65b1e76..5b831ee 100644
Binary files a/node_modules/dmg-builder/vendor/mac_alias/alias.pyc and b/node_modules/dmg-builder/vendor/mac_alias/alias.pyc differ
diff --git a/node_modules/dmg-builder/vendor/mac_alias/bookmark.pyc b/node_modules/dmg-builder/vendor/mac_alias/bookmark.pyc
index bc51c70..09f0f99 100644
Binary files a/node_modules/dmg-builder/vendor/mac_alias/bookmark.pyc and b/node_modules/dmg-builder/vendor/mac_alias/bookmark.pyc differ
diff --git a/node_modules/dmg-builder/vendor/mac_alias/osx.py b/node_modules/dmg-builder/vendor/mac_alias/osx.py
index d4af65d..b43fa3a 100644
--- a/node_modules/dmg-builder/vendor/mac_alias/osx.py
+++ b/node_modules/dmg-builder/vendor/mac_alias/osx.py
@@ -421,11 +421,17 @@ _fgetattrlist = libc.fgetattrlist
 _fgetattrlist.argtypes = [c_int, POINTER(attrlist), c_void_p, c_ulong, c_ulong]
 _fgetattrlist.restype = c_int
 
-_statfs = libc['statfs$INODE64']
+try:
+     _statfs = libc['statfs$INODE64']
+ except (KeyError, AttributeError):
+     _statfs = libc['statfs']
 _statfs.argtypes = [c_char_p, POINTER(struct_statfs)]
 _statfs.restype = c_int
 
-_fstatfs = libc['fstatfs$INODE64']
+try:
+     _fstatfs = libc['fstatfs$INODE64']
+ except (KeyError, AttributeError):
+     _fstatfs = libc['fstatfs']
 _fstatfs.argtypes = [c_int, POINTER(struct_statfs)]
 _fstatfs.restype = c_int
 
diff --git a/node_modules/dmg-builder/vendor/mac_alias/osx.pyc b/node_modules/dmg-builder/vendor/mac_alias/osx.pyc
index 6499fc8..6ca6126 100644
Binary files a/node_modules/dmg-builder/vendor/mac_alias/osx.pyc and b/node_modules/dmg-builder/vendor/mac_alias/osx.pyc differ
diff --git a/node_modules/dmg-builder/vendor/mac_alias/utils.pyc b/node_modules/dmg-builder/vendor/mac_alias/utils.pyc
index fb6c281..a94124e 100644
Binary files a/node_modules/dmg-builder/vendor/mac_alias/utils.pyc and b/node_modules/dmg-builder/vendor/mac_alias/utils.pyc differ
