diff --git a/node_modules/electron-builder/out/builder.js b/node_modules/electron-builder/out/builder.js
index 59383a1..0ab0d54 100644
--- a/node_modules/electron-builder/out/builder.js
+++ b/node_modules/electron-builder/out/builder.js
@@ -69,10 +69,6 @@ function normalizeOptions(args) {
 
   function processTargets(platform, types) {
     function commonArch(currentIfNotSpecified) {
-      if (platform === _appBuilderLib().Platform.MAC) {
-        return args.x64 || currentIfNotSpecified ? [_builderUtil().Arch.x64] : [];
-      }
-
       const result = Array();
 
       if (args.x64) {
@@ -242,7 +238,8 @@ function createTargets(platforms, type, arch) {
   const targets = new Map();
 
   for (const platform of platforms) {
-    const archs = platform === _appBuilderLib().Platform.MAC ? [_builderUtil().Arch.x64] : arch === "all" ? [_builderUtil().Arch.x64, _builderUtil().Arch.ia32] : [(0, _builderUtil().archFromString)(arch == null ? process.arch : arch)];
+    const archs = arch === "all" && platform !== _appBuilderLib().Platform.MAC ? [_builderUtil().Arch.x64, _builderUtil().Arch.ia32] : [(0, _builderUtil().archFromString)(arch == null ? process.arch : arch)];
+     const archToType = new Map();
     const archToType = new Map();
     targets.set(platform, archToType);
 
